### DISCUSSION TITLE
Integrate App with Google Analytics eCommerce Events Tracking

### DISCUSSION THREAD

**Discussion by hamadatalaat** [inspireui/support#5636] [Working on Features]
I think it is an ecommerce app to make it fully functionable it should be ready to integrate with google analytics especially with the advance ecommerce feature so admins of woocommerce should get reports in google analytics for eCommerce events and use it also in Google Tag Manager i think these reports it is very important and complete the online commerce cycle. Thanks

**Comment by osman-kocak** [comment]
its very needed

**Comment by tryabp** [comment]
+1

**Comment by ameenbadri7** [comment]
one of my customers wanted this on fluxstore and then I made it by changing the flutter fluxstore by making custom events.
I guess fluxstore can make this feature optional.

BTW google analytics is working out of the box with fluxstore but with the default settings.

**Comment by simonpham** [comment]
Event tracking using Firebase Analytics is available in FluxStore 3.4.0
